task/06-conditions-n-loops-tasks.js

```diff
@@ -296,7 +385,20 @@ function getDigitalRoot(num) {
  *   '{[(<{[]}>)]}' = true 
  */
 function isBracketsBalanced(str) {
-    throw new Error('Not implemented');
+    let stack = [];
+    let opening_brackets = ['[','{','(','<'];
+    let closing_brackets = [']','}',')','>'];
+    for(let i = 0; i < str.length; i++){
+        if(opening_brackets.indexOf(str[i])>=0){
+            stack.push(opening_brackets.indexOf(str[i]));
+        }
+        else{   
+            if(closing_brackets[stack.pop()] != str[i]){
```

Comment:
operator !== is always preferable
sometimes ''!=' may cause an error
(check at console :
"0" != 0
and "0" !== 0)